### PR TITLE
Add troubleshooting for Roomba integration

### PR DESCRIPTION
**Pull request by andeart**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Added troubleshooting section for iRobot Roomba and Braava integration.
	- Provided guidance for resolving connection issues during setup.
	- Recommended steps include ensuring the device is actively cleaning and potential factory reset if problems persist.

### CODE REVIEW

File: source/_integrations/roomba.markdown [hunk 1]
```diff
@@ -88,3 +88,9 @@ Use this credentials in dorita980 lib :)
 ```
 
 Copy the password (everything between `=>` and `<=`, not including leading and trailing whitespace) into the Home Assistant password dialog.
+
+## Troubleshooting
+
+- **Integration wizard shows "Failed to connect" after submitting the password**: Before attempting a factory reset (which can be a cumbersone process), attempt submitting the password in the integration wizard while the Roomba is actively running (i.e. cleaning). Avoid opening the app to start a manual job to help with this. Instead, push the physical clean button on the device directly to start the manual job. This appears to resolve the issue on some models because they answer queries only while actively running ([Github thread](https://github.com/home-assistant/core/issues/117071#issuecomment-2295765878)).
```
Comment (frenck) [inline source/_integrations/roomba.markdown]: Let's remove the reference to GitHub issues. It shouldn't be needed. The documentation described all there is to it.

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/roomba.markdown (hunk 1 above)
Comment (andeart) [reply source/_integrations/roomba.markdown]: That makes sense. I'll push an update.

**Review by frenck — APPROVED** [review]
Thanks, @andeart 👍 

Happy New Year 🥂 

../Frenck

**Comment by andeart** [comment]
Thanks for the quick review, @frenck . It's less than an hour away from 2025 where you are, I believe. I hope you have an excellent new year too!

**Comment by frenck** [comment]
Hehe I'm going to log off to pop some champagne now 🍾 

../Frenck

**Pull request merged**